legacy/src/api/assets/asset-transfers.md

```diff
@@ -102,7 +111,8 @@ Transfer an asset to a recipient. Some assets can be transfered only in whole
   "recipientAlias": "+64212312345",
   "createdAt": "2020-05-01T12:30:00.000Z",
   "updatedAt": "2020-05-02T01:03:37.222Z",
-  "suppressNotifications": false
+  "suppressNotifications": false,
+  "url": "https://app.centrapay.com/transfer/M7Kn2stAxNa6ri7h"
```

Comment:
Suggestion: Use 2 examples, we'd never return recipient fields and url together, but not sure how strict we are around this...

Reply:
I don't mind separating out the responses :)